clconf/clconf.go

```diff
@@ -22,6 +23,60 @@ import (
 // Splitter is the regex used to split YAML_FILES and YAML_VARS
 var Splitter = regexp.MustCompile(`,`)
 
+// ConfSources contains sources of yaml for loading. See Load() for more info
+type ConfSources struct {
+	Environment bool     // Environment loads config from environment vars
+	Files       []string // Filenames to read
+	Overrides   []string // Base64 encoded strings of yaml
```

Comment:
I think you should use the normal comment style here.  As suffix comments i don't think they would be provided by the IDE...  Should probably be:

```golang
type ConfSources struct {
    // Environment loads config from environment variables if true
    Environment bool
    // Files is a list of yaml files to load
    Files        []string
    // Overrides is a list of base64 encoded yaml strings to load
    Overrides
...
```

Reply:
TIL.